lib/commands/os/configure.ts

```diff
@@ -279,37 +280,23 @@ export default class OsConfigureCmd extends Command {
 					};
 				}),
 			);
-			const imagefs = await import('resin-image-fs');
+
+			const imagefs = await import('balena-image-fs');
 
 			for (const { name, content } of files) {
-				await imagefs.writeFile(
-					{
-						image,
-						partition: BOOT_PARTITION,
-						path: path.join(CONNECTIONS_FOLDER, name),
-					},
-					content,
-				);
+				await imagefs.interact(image, BOOT_PARTITION, async (_fs) => {
+					return await promisify(_fs.writeFile)(
+						path.join(CONNECTIONS_FOLDER, name),
+						content,
+					);
+				});
 				console.info(`Copied system-connection file: ${name}`);
 			}
 		}
 	}
 }
 
 async function validateOptions(options: FlagsDef) {
-	if (process.platform === 'win32') {
-		throw new ExpectedError(stripIndent`
-			Unsupported platform error: the 'balena os configure' command currently requires
-			the Windows Subsystem for Linux in order to run on Windows. It was tested with
-			the Ubuntu 18.04 distribution from the Microsoft Store. With WSL, a balena CLI
-			release for Linux (rather than Windows) should be installed: for example, the
-			standalone zip package for Linux. (It is possible to have both a Windows CLI
-			release and a Linux CLI release installed simultaneously.) For more information
-			on WSL and the balena CLI installation options, please check:
-			- https://docs.microsoft.com/en-us/windows/wsl/about
-			- https://github.com/balena-io/balena-cli/blob/master/INSTALL.md
-		`);
-	}
```

Comment:
Nice 👍  This PR could also delete a similar note in the `INSTALL-WINDOWS.md` file:
https://github.com/balena-io/balena-cli/blob/v12.43.0/INSTALL-WINDOWS.md#balena-os-configure